CHANGELOG.md

```diff
@@ -1,10 +1,30 @@
 # Fluidcontent Change log
 
+upcoming
+------------------
+
+- :exclamation: Support of [TYPO3 6.2 dropped](https://github.com/FluidTYPO3/fluidcontent/commit/f1df9a4b7ee563335c8f1feeb7925e6f820a7f19)
```

Comment:
We will use the same name but I'm not sure yet if we will reuse or recreate it. The link will be the same in the end.